circle.yml

```diff
@@ -65,3 +65,8 @@ deployment:
     owner: mozilla
     commands:
       - bin/circleci-trigger-jenkins.sh
+  release:
+    tag: /\d+-\d+-\d+(\.\d+)?/
```

Comment:
I was actually curious if we could just add this line to the `jenkins` section. But this should definitely work.

We could also get more specific `tag: /\d{4}-\d{2}-\d{2}(\.\d)?/`, but that's not really necessary either (that's what I have in basket).